packages/react-scripts/fixtures/kitchensink/integration/env.test.js

```diff
@@ -6,7 +6,15 @@ describe('Integration', () => {
     it('file env variables', async () => {
       const doc = await initDOM('file-env-variables')
 
-      expect(doc.getElementById('feature-file-env-variables').textContent).to.equal('fromtheenvfile.')
```

Comment:
You mean in the `expect`?
I usually insert them as an extra caution to ensure that the expected string is placed exactly where I want to.

Is it extremely necessary? No :P